R-package/revdep/email.yml

```diff
@@ -0,0 +1,5 @@
+release_date: ???
```

Comment:
Do we need these filled in?

Reply:
I think only if you're mass emailing package maintainers with downstream dependencies? Which we don't have at this point. 

Also worth noting this `revdep` library is still in an experimental state. I was just using it to quickly see if we had any downstream dependencies yet.